Project.toml

```
@@ -16,7 +16,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 CEnum = "0.4, 0.5"
 MPI = "0.20"
 MPIPreferences = "0.1.3"
-P4est_jll = "=2.8.1"
+P4est_jll = "=2.8.1, 2"
```

```suggestion
P4est_jll = "=2.8.7"
```
Looks like v2.8.7 is out now. Let's see if CI passes again.